### PR TITLE
Is refundable update

### PR DESCRIPTION
**Pull request by Klongrich**
the "minor fixes" commit is just updating what is returned in the "useEffect()" in Claimer.tsx and a "," in MainApp.tsx

### CODE REVIEW

File: src/components/MainApp.tsx [hunk 1]
```diff
@@ -93,6 +98,10 @@ const MainApp: React.FC<IMainApp> = ({ address, web3, onConnect, meta }) => {
     interval: 10000
   };
 
+  function get_isRasied() {
```
Comment (altbee) [inline src/components/MainApp.tsx]: Can you pls remove this empty function?

File: src/components/Claimer.tsx [hunk 2]
```diff
@@ -180,7 +271,7 @@ const Claimer: React.FC<IClaimer> = ({
             <Text fontSize="18px" color="lid.fg">
               {`More ${meta.tokenSymbol} available to claim in`}
             </Text>
-            <CountDownShort expiryTimestamp={claimPeriod * 1000} />
```
Comment (altbee) [inline src/components/Claimer.tsx:183]: Did you test this line? The expiryTimestamp should be in milliseconds, I think.

File: src/components/Claimer.tsx [hunk 3]
```diff
@@ -66,6 +93,70 @@ const Claimer: React.FC<IClaimer> = ({
...
+          w="100%"
+          mb="20px"
+          p="20px"
+        >
+          <Text fontSize={['24px', '36px']} fontWeight="bold">
+            {`Claim Refund`}
+          </Text>
+    
+          <Input
+            fontSize="18px"
```
Comment (altbee) [inline src/components/Claimer.tsx:124]: Not sure you just followed the design, but I think we don't need the input box for refund address.
We can just use the address connected to your dapp and in this case you can move getRefundableEth function call to the MainApp.tsx using multicalljs

**Review by altbee — CHANGES_REQUESTED** [review]

File: src/components/Claimer.tsx (hunk 2 above)
Comment (Klongrich) [reply src/components/Claimer.tsx:183]: I did not add that line.

Comment (Klongrich) [reply src/components/Claimer.tsx:183]: Can double check though

File: src/components/Claimer.tsx (hunk 3 above)
Comment (Klongrich) [reply src/components/Claimer.tsx:124]: Yeah I was told to make it so they have to enter an address. Can change it to just use the connect wallet though.

Comment (altbee) [reply src/components/Claimer.tsx:124]: Ok then let's leave it for now

File: src/components/Claimer.tsx (hunk 2 above)
Comment (altbee) [reply src/components/Claimer.tsx:183]: but I can see this line is changed

**Pull request merged**